### DISCUSSION TITLE
iOS Share Extension

### DISCUSSION THREAD

**Discussion by emilkaiser** [ionic-team/capacitor#5087] [Q&A]
I was able to get an iOS Share Extension working with Capacitor. I hade to set `Require Only App-Extension-Safe API` to `No` in order for the app to compile. As I understand it, [this will cause the app to be rejected by Apple when reviewing](https://developer.apple.com/library/archive/documentation/General/Conceptual/ExtensibilityPG/ExtensionScenarios.html). 

The reason seems to be the CapacitorCordova pod and `'sharedApplication' is unavailable: not available on iOS (App Extension) - Use view controller based solutions where appropriate instead`.

I don't know enough about the underlying technology to know if it is even possible, but it would be really cool if you could build a Capacitor Share Extension and get it through Apple reviews. What are your thoughts on this?

**Comment by connorlindsey** [comment]
Hey @emilkaiser, I'd also be very interested in this. I'm currently experimenting with Capacitor for a share extension (will try RN next, then just resort to building a native app). 

Would you be willing to share a bit more about what you did to get the share extension working?
